source/MaterialXTest/ShaderValid.cpp

```diff
@@ -45,7 +45,7 @@ namespace mx = MaterialX;
 
 #define LOG_TO_FILE
 
-extern void loadLibraries(const mx::StringVec& libraryNames, const mx::FilePath& searchPath, mx::DocumentPtr doc, 
+extern void loadLibraries(const mx::StringVec& libraryNames, const mx::FilePath& searchPath, mx::DocumentPtr doc,
```

Comment:
Minor: Strange you get these format changes all the time. Is it because your using notepad++ or such that your spacing/tabbing differs?